### ISSUE TITLE
Dialect: DuckDB "LOAD spatial;" causes indentation rules to not be enforced 

### ISSUE THREAD

**Issue by gregorywaynepower** [quarylabs/sqruff#882]
Hey folks! I'm currently using sqruff 0.19.1 installed via cargo on Windows 10 working with the DuckDB dialect. For some reason a `LOAD spatial;` is throwing off the rest of my linting rules.

```
LOAD spatial;

-- SUMMARIZE cpd_incidents;
-- DESCRIBE cpd_incidents;

SELECT location FROM cpd_incidents LIMIT 1;

SELECT
    crimeday AS "Crime Day",
    count(crimeday) AS "Crime Count"
FROM
    cpd_incidents
WHERE
    crimeday IS NOT NULL
GROUP BY
    crimeday
ORDER BY
    "Crime Count" DESC;
```
Becomes
```
LOAD spatial ;

-- SUMMARIZE cpd_incidents;
-- DESCRIBE cpd_incidents;

SELECT location FROM cpd_incidents LIMIT 1 ;

SELECT
crimeday AS "Crime Day",
count (crimeday) AS "Crime Count"
FROM
cpd_incidents
WHERE
crimeday IS NOT NULL
GROUP BY
crimeday
ORDER BY
"Crime Count" DESC ;
```
When I remove the LOAD spatial;  my linting rules are parsed properly.
```
-- LOAD spatial;

-- SUMMARIZE cpd_incidents;
-- DESCRIBE cpd_incidents;

SELECT location FROM cpd_incidents LIMIT 1;

SELECT
    crimeday AS "Crime Day",
    count(crimeday) AS "Crime Count"
FROM
    cpd_incidents
WHERE
    crimeday IS NOT NULL
GROUP BY
    crimeday
ORDER BY
    "Crime Count" DESC;

```

**Comment by benfdking** [comment]
This is probably because at the moment we don't support the load syntax.

**Comment by gregorywaynepower** [comment]
That's valid--it's very much a feature that has seen its fare share of changes in DuckDB.

**Comment by benfdking** [comment]
Load is fixed and pushed. I'll do `SUMMARIZE` and `DESCRIBE` next.

**Comment by gregorywaynepower** [comment]
Thanks @benfdking!

**Comment by benfdking** [comment]
Those are fixed and being pushed in release `0.21.3`

**Comment by benfdking** [comment]
Hey @gregorywaynepower, I am going to close this. Let us know if you have any issues.
